GameDB: NBA 2K6 fix hangs on PAL version

bslenul opened the pull request:
The game uses a different CRC depending on the language and a different address for the patch.

Mrlinkwii reviewed (approved):
assumed already tested and working

bslenul commented:
Yup, switched my BIOS language, booted the game, noted the CRC, looked up for the correct address with Cheat Engine, modified the game database file and tested, rince and repeat for all the supported languages.
Not tested extensively tho, I just checked if it booted and if I was able to start a match by selecting "Quick Game".
